app-modules/service-management/src/Filament/Resources/ServiceRequestTypeResource/Pages/ServiceRequestTypeEmailTemplatePage.php

```diff
@@ -0,0 +1,149 @@
+class ServiceRequestTypeEmailTemplatePage extends EditRecord
+{
+    protected static string $resource = ServiceRequestTypeResource::class;
+
+    #[Locked]
+    public ?string $type;
+
+    public static ?string $navigationGroup = 'Email Templates';
+
+    public function getRelationManagers(): array
+    {
+        // Needed to prevent Filament from loading the relation managers on this page.
+        return [];
+    }
+
+    public function form(Form $form): Form
+    {
+        return $form
+            ->schema([
+                Section::make()
+                    ->columns()
+                    ->schema([
+                        TiptapEditor::make('subject')
+                            ->label('Subject')
+                            ->placeholder('Enter the email subject here...')
+                            ->rules(['required'])
+                            ->extraInputAttributes(['style' => 'min-height: 2rem; overflow-y:none;'])
+                            ->disableToolbarMenus()
+                            ->mergeTags([
+                                'created',
+                                'updated',
+                                'status',
+                                'assigned to',
+                                'title',
+                                'type',
+                            ])
+                            ->showMergeTagsInBlocksPanel(false)
+                            ->helperText('You may use “merge tags” to substitute information about a service request into your subject line. Insert a “{{“ in the subject line field to see a list of available merge tags')
+                            ->columnSpanFull(),
+                        TiptapEditor::make('body')
+                            ->label('Body')
+                            ->placeholder('Enter the email body here...')
+                            ->rules(['required'])
+                            ->extraInputAttributes(['style' => 'min-height: 12rem;'])
+                            ->mergeTags([
+                                'created',
+                                'updated',
+                                'status',
+                                'assigned to',
+                                'title',
+                                'type',
+                            ])
+                            ->columnSpanFull(),
+                    ]),
+            ]);
+    }
+
+    public function save(bool $shouldRedirect = true, bool $shouldSendSavedNotification = true): void
+    {
+        $data = $this->form->getState();
+        $data['service_request_type_id'] = $this->getRecord()->id;
+        $data['type'] = $this->type;
+
+        if ($this->template) {
+            $this->template->update($data);
+        } else {
+            $this->getRecord()?->templates()->create($data);
+        }
+
+        unset($this->template);
+
+        parent::save($shouldRedirect, $shouldSendSavedNotification);
```

Comment:
Instead of calling `save()`, maybe just call the part of the method that sends the notification? As we don't want the service request type to be saved